.rubocop.yml

```diff
@@ -164,3 +164,7 @@ Rails/ActionControllerFlashBeforeRender:
 
 RSpec/FilePath:
   Enabled: false
+
+# Occasionally it's useful to create big lists
+FactoryBot/ExcessiveCreateList:
+  Enabled: false
```

Comment:
**nit** - might be worth an explicit `Exlude` for the individual files here if possible